includes/helper/class-wp-job-manager-helper.php

```diff
@@ -504,10 +504,28 @@ public function licence_output() {
 		if ( ! empty( $_POST ) ) {
 			$this->handle_request();
 		}
-		$licenced_plugins = $this->get_installed_plugins();
+		$licenced_plugins   = $this->get_installed_plugins();
+		$show_bulk_activate = $this->show_bulk_activation_form( $licenced_plugins );
 		include_once dirname( __FILE__ ) . '/views/html-licences.php';
 	}
 
+	/**
+	 * Return if we should show or not the bulk activation form.
+	 *
+	 * @param array $licenced_plugins The list of licensed plugins to handle.
+	 *
+	 * @return bool If we should show the bulk activation form or not.
+	 */
+	private function show_bulk_activation_form( $licenced_plugins ) {
+		foreach ( array_keys( $licenced_plugins ) as $product_slug ) {
+			$licence = self::get_plugin_licence( $product_slug );
+			if ( empty( $licence['licence_key'] ) && apply_filters( 'wpjm_display_license_form_for_addon', true, $product_slug ) ) {
```

Comment:
Maybe we should add the hook/filter documentation?

Reply:
This filter isn't new, it was added in #2332, and as far as I know it is intended to be kinda of "secret". I can document it, sure, but honestly I'm not sure if that's something we really want to do. :thinking: